.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: python
 python:
-  - "2.7"
+  - "3.7"
```

Comment:
We run on 3.6 internally right now and there are some language changes in 3.7 like extra reserved keywords and breaking stdlib changes. Maybe we should test on both 3.6 and 3.7 for compatibility reasons?